pkg/provider-local/controller/backupentry/actuator.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package backupentry
+
+import (
+	"context"
+	"path/filepath"
+
+	"github.com/gardener/gardener/extensions/pkg/controller/backupentry/genericactuator"
+	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const BackupPath = "backupPath"
```

Comment:
```suggestion
const BackupPath = "hostPath"
```

Since that is how it is searched for in the etcd-druid. I wonder if there is anyway to expose this as a constant that is used by both this controller and etcd-druid.

Reply:
the `etcd-backup` secret already have `bucketName` I think it would be more consistent to leave it as `bucketPath`. For the constant, I agree and I will add it in the `Etcd-druid` PR, so once it is revendorded we can use it.